pkg/sql/alter_table.go

```diff
@@ -297,7 +297,14 @@ func (n *alterTableNode) startExec(params runParams) error {
 				if containsThisColumn {
 					if containsOnlyThisColumn || t.DropBehavior == tree.DropCascade {
 						if err := params.p.dropIndexByName(
-							params.ctx, tree.UnrestrictedName(idx.Name), n.tableDesc, false, t.DropBehavior, ignoreOutboundFK,
+							params.ctx,
+							tree.UnrestrictedName(idx.Name),
+							n.tableDesc,
+							false, /* ifExists */
+							nil,   /* constraints */
+							true,  /* droppingColumn */
+							t.DropBehavior,
+							ignoreOutboundFK,
```

Comment:
nit: `ignoreOutboundFK` is the same thing as `droppingColumn` -- we just used a self-doc'ing enum over an inline-commented bool.